testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js

```diff
@@ -1469,6 +1469,49 @@ QUnit.module('regressions', {
         assert.strictEqual(selectionChangedStub.lastCall.args[0].selectedItem, null);
     });
 
+    QUnit.test('onSelectionChanged event should trigger on item selection if its text is equal to input text (T991350)', function(assert) {
```

Comment:
Let's add a new submodule with instance configuration in `beforeEach` to minimize copypaste?